packages/fxa-auth-server/lib/senders/emails/partials/paymentPlanDetails/index.mjml

```diff
@@ -0,0 +1,32 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-text css-class="text-body">
+  <span><b data-l10n-id="payment-details">Payment details:</b></span>
```

Comment:
In the old template, this text looks larger than it is in the new template.

Also you can remove the `span` this is in.

Reply:
I have the old template screenshot above the new one (lmk if I should have the new one first moving forward). 👀